src/plugins/intel_cpu/src/nodes/common/cpu_convert.cpp

```diff
@@ -518,6 +518,40 @@ bool isConversionTruncatesRange(const Precision & from, const Precision & to) {
     INTEL_CPU_CVT(FP32, FP32), INTEL_CPU_CVT(FP16, FP16), INTEL_CPU_CVT(BF16, BF16), INTEL_CPU_CVT(FP64, FP64), \
     INTEL_CPU_CVT(BOOL, BOOL)
 
+#define INTEL_CPU_CVT_FROM_BIN(DT) OV_CASE(Precision::DT, PrecisionInfo<Precision::DT>::value_type)
+
+#define INTEL_CPU_CVT_FROM_BIN_LIST                                                                 \
+    INTEL_CPU_CVT_FROM_BIN(FP32), INTEL_CPU_CVT_FROM_BIN(FP16), INTEL_CPU_CVT_FROM_BIN(BF16),       \
+    INTEL_CPU_CVT_FROM_BIN(FP64), INTEL_CPU_CVT_FROM_BIN(I16), INTEL_CPU_CVT_FROM_BIN(U8),          \
+    INTEL_CPU_CVT_FROM_BIN(I8), INTEL_CPU_CVT_FROM_BIN(U16), INTEL_CPU_CVT_FROM_BIN(I32),           \
+    INTEL_CPU_CVT_FROM_BIN(U32), INTEL_CPU_CVT_FROM_BIN(I64), INTEL_CPU_CVT_FROM_BIN(U64),          \
+    INTEL_CPU_CVT_FROM_BIN(BOOL)
+
+struct ConvertFromBinContext {
+    const void *srcPtr;
+    void *dstPtr;
+    size_t size;
+    bool converted;
+};
+
+template<typename T>
+struct ConvertFromBinPrecision {
+    void operator()(ConvertFromBinContext &ctx) {
+        auto src = static_cast<const uint8_t *>(ctx.srcPtr);
+        auto dst = static_cast<T *>(ctx.dstPtr);
+        const size_t nBits = 8;
+        const size_t nBytes = ctx.size / nBits + (ctx.size % nBits != 0 ? 1 : 0);
```

Comment:
I guess you can use [rnd_up](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/utils/general_utils.h#L17) utility. Just slightly improves readability.

Reply:
Thank you for the hint.